🫵🏽 Implement company type product lists rest api contracts

daniel-rose opened the pull request:
- implement contracts for expanding rest product list update request
- add expand plugin to expand rest product list update request
- ...

sourcecube reviewed (approved):
The title doesn't represent the content. (;

daniel-rose committed:
🫵🏽 Implement company type product lists rest api contracts
- implement contracts for expanding rest product list update request
- add expand plugin to expand rest product list update request
...

daniel-rose committed:
Add phpcs fixes
